openlibrary/core/cache.py

```diff
@@ -314,19 +314,19 @@ def get(self, key):
         stats.begin("memcache.get", key=key)
         value = self.memcache.get(key)
         stats.end(hit=value is not None)
-        return value and simplejson.loads(value)
+        return value and json.loads(value)
```

Comment:
I do not love this approach for a return statement because we (and the caller) are clueless about what the return type is going to be...  If value is '' then we return a str, if If value is 0 then we return an int, If value is 0.0 then we return a float, If value is {} then we return a dict, etc.

What about `return value and json.loads(value) or {}` (we always return a dict) or `return value and json.loads(value) or None` (we return an Optional[dict])?

Reply:
I'm a bit hesitant to change this line since this is used in a few places.   Perhaps we can open a new issue for this refactor?

Reply:
OK.  Please open that issue.  It will make more sense when we can add Python type hints.